docusaurus.config.js

```diff
@@ -285,6 +285,13 @@ const config = {
           //     },
           //   ],
           // },
+          {
```

Comment:
I think this should be swapped with GitHub as it looks a bit odd:

![image](https://github.com/opentofu/opentofu.org/assets/86970079/6eca6bd2-54f3-47fd-8abb-3462c2193c27)

Reply:
Good catch. Updated now
![Screenshot 2024-01-08 051720](https://github.com/opentofu/opentofu.org/assets/62661/31d6a954-c8ab-4b38-89b0-5bdf82819335)